types/replicatedservice/v1/replicatedservice.py

```diff
@@ -41,9 +41,9 @@ def GenerateConfig(context):
       'properties': {
           'apiVersion': 'v1',
           'kind': 'Service',
-          'namespace': namespace,
           'metadata': {
               'name': service_name,
+              'namespace': namespace,
```

Comment:
Yes. The api paths include the namespace, but in the configuration, the namespace lives in the metadata section. See, for example https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/skydns-svc.yaml.in.